fastly/ngwaf/v1/workspaces/api_create.go

```diff
@@ -19,18 +19,22 @@ type AttackSignalThresholdsCreateInput struct {
 // CreateInput specifies the information needed for the Create() function to
 // perform the operation.
 type CreateInput struct {
+	// AttackSignalThresholds are the parameters for system site alerts.
+	AttackSignalThresholds *AttackSignalThresholdsCreateInput `json:"attack_signal_thresholds,omitempty"`
+	// ClientIPHeaders lists the request headers containing the client IP address.
+	ClientIPHeaders []string `json:"client_ip_headers"`
 	// Context, if supplied, will be used as the Request's context.
 	Context *context.Context `json:"-"`
-	// Name is the name of a workspace to create (required).
-	Name *string `json:"name"`
+	// DefaultBlockingResponseCode is the default response code.
+	DefaultBlockingResponseCode *int `json:"default_blocking_response_code"`
 	// Description is the description of a workspace.
 	Description *string `json:"description"`
-	// Mode is the mode of a workspace.
-	Mode *string `json:"mode"`
-	// AttackSignalThresholds are the parameters for system site alerts.
-	AttackSignalThresholds *AttackSignalThresholdsCreateInput `json:"attack_signal_thresholds,omitempty"`
 	// IPAnonymization is the selected option to anonymize IP addresses.
 	IPAnonymization *string `json:"ip_anonymization"`
+	// Mode is the mode of a workspace.
+	Mode *string `json:"mode"`
+	// Name is the name of a workspace to create (required).
+	Name *string `json:"name"`
```

Comment:
Besides adding the two new fields I changed the order to be alphabetic. Sorry for the confusion!